documentation/document-api-guide.html

```diff
@@ -183,6 +183,11 @@ <h3 id="field-path-updates">Field Path Updates</h3>
 The <em>where clause</em> decides whether the document is to be changed at all,
 and also sets variables to use in the field path
 when deciding which parts of the document to change.
+Field path updates are only supported on non-attribute
+<a href="reference/search-definitions-reference.html#field">fields</a>,
+<a href="reference/search-definitions-reference.html#index">index</a> fields,
```

Comment:
If a field is both an indexed field and an attribute, then the document is updated in the document store, the index is updated, but the attribute is not updated. Thus you can get old values in document summary requests and old values being used in ranking and grouping.